README.md

```diff
@@ -29,146 +31,73 @@ genesis init --kit prometheus
-  `secret/us/west/bosh/users/admin:password`.
-- **params.bosh_ca_cert** - A vault path which contains the CA certificate for the CA that signed the BOSH
-  monitored director's SSL certificate. For example: `secret/us/west/prod/bosh/ssl/ca:certificate`.
-
-#### blackbox
-- **params.monitored_endpoints** - A list of URLs that Prometheus should monitor
-  with its [blackbox_exporter][2]. These will show up in the `Probe: HTTPS Summary` dashboard
-- **params.skip_ssl_verify** - Defines whether the Promtheus exporters skip SSL cert
-  validation when connecting to the various monitored components they watch. Off by default
-  (certs are checked). Enable this if you have self-signed certs in your deployment.
-
-#### postgres
-
-Default values of the `postgres` subkit are set up to work nicely with the database instance
-provided in the [cf-genesis-kit][3] deployment.
-
-- **params.pg_host** - The IP of the monitored Postgres instance
-- **params.pg_port** - The port that the monitored Postgres is listening on (defaults to 5432)
-- **params.pg_sslmode** - Determines whether SSL is used or not when connecint go the monitored
-  postgres (default is 'disabled')
-- **params.pg_user** - The user to connect to the monitored Postgres instance with (defaults to `shield).
-- **params.pg_password** - A vault path which contains the password for the above PG user. For example:
-  `secret/us/west/cf/postgres:shield_password`
-
-#### elasticsearch
-
-- **params.elasticsearch_uri** - The URL for the monitored ElasticSearch cluster's master node
-
-#### rabbitmq
-
-Default values of the `rabbitmq` subkit are set up to work well with the [cf-rabbitmq-genesis-kit][6].
-
-- **params.rabbitmq_url** - The URL for the monitored RabbitMQ cluster (e.g. `rabbitmq.system.bosh-lite.com`)
-- **params.rabbitmq_user** - The username used to connect to the monitored RabbitMQ cluster
-  ( defaults to `admin`)
-- **params.rabbitmq_password** - A vault path which contains the password for the above
-  RabbitMQ user. For example: `secret/us/west/cf-rabbitmq/rabbitmq/client_secrets:admin`
-- **params.rabbitmq_include_queues** - A regex of queues to explicitly include in monitoring
-- **params.rabbitmq_exclude_queues** - A regex of queues to explicitly exclude in monitoring
+### General Infrastructure Configuration
+* `disk_type` - The `persistent_disk_type` that Prometheus should use
+  for storage. (default: `prometheus`)
+* `vm_type`- The `vm_type` that Prometheus should be deployed on.
+  (default: `default`) 
+* `network` - The `network` that Prometheus should be deployed on.
+  (default: `prometheus`)
+* `stemcell_os` - The operating system stemcell you want to deploy on.
+  (default: `ubuntu-trusty`)
+* `stemcell_version` - The specific version of the stemcell you want
+  to deploy on. (default: `latest`)
+* `static_ip` - The static IP to assign to the VM. (no default)
+
+### Minio Related Configuration
```

Comment:
Minio?

Reply:
Prometheus, or as I like to call it, Minio.

Reply:
Taking minnie to promie this year?